mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -207,19 +241,26 @@ def StartObjectDetectionTask(self, taskId=None, systemState=None, visionTaskPara
         return self._ExecuteCommand(command, timeout=timeout)
 
     def StartContainerDetectionTask(self, taskId=None, systemState=None, visionTaskParameters=None, timeout=2.0, **ignoredArgs):
-        # type: (Optional[types.SystemState], Optional[types.visionTaskContainerDetectionParametersSchema], float, Dict) -> Optional[Dict]
+        # type: (Optional[str], Optional[Union[types.SystemState, Dict]], Optional[Union[types.visionTaskContainerDetectionParametersSchema, Dict]], float, Any) -> Optional[Dict[str, str]]
         """Starts container detection thread to continuously detect a container. the vision server will send detection results directly to mujin controller.
 
         Args:
-            taskId (str, optional): If specified, the specific taskId to use
-            systemState (types.SystemState or dict): The state of the system. Used to select the profile that the vision task will use. See "Profile Selection" documentation for more details.
-            visionTaskParameters (types.visionTaskContainerDetectionParametersSchema or dict): Parameters for the container detection task. These take precedence over the base profile selected via the system state, but are overwritten by the overwrite profile.
+            taskId (str, optional): If specified, the specific taskId to use.
+            systemState (types.SystemState | dict, optional): The state of the system. Used to select the profile that the vision task will use. See "Profile Selection" documentation for more details.
```

Comment:
I don't think the syntax is understood when the doc is generated.

Reply:
The docs with this syntax are generated the same way as without it. The type in parentheses is displayed as is in both cases.

While the parameter type is taken from the type comment. E.g.:
```
StartObjectDetectionTask(taskId: Optional[str] = None, systemState: Optional[Union[mujinvisiontypes.SystemState, Dict]] = None, visionTaskParameters: Optional[Union[mujinvisiontypes.visionTaskObjectDetectionParametersSchema, Dict]] = None, timeout: float = 2.0, **ignoredArgs: Any)→ Optional[Dict[str, str]]
```

But taking into account your comment above, it doesnt matter, lets remove the dict part.